### PR TITLE
View Exam Detail

### PR DESCRIPTION
**Pull request by Phvanhoang**
image
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/35520)

## WHAT
- View an exam's detail

## Evidence 
image

### CODE REVIEW

**Comment by Phvanhoang** [comment]
Ready

File: app/views/supervisor/exams/_exam_question.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<li class="exam-questions">
+  <%= question.content %>
+  <ol class="exam-question-answers">
+    <% question.answers.each do |answer| %>
```
Comment (vanvtt-0952) [inline app/views/supervisor/exams/_exam_question.html.erb:4]: có bị N+1 query ko em

Comment (Phvanhoang) [reply app/views/supervisor/exams/_exam_question.html.erb:4]: Dạ không bị N+1 query ạ
image

**Comment by Phvanhoang** [comment]
Ready

**Pull request merged**